plugins/blur/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(SOURCES
     blur.cpp)
 
 if (APPLE)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
```

Comment:
For Linux spesific stuff we can do `if(CMAKE_SYSTEM_NAME MATCHES "Linux")`

A lot of the current Linux checks would work for Linux/BSD.

I'm mainly aiming to add in Linux without re-arranging code-blocks (which makes review harder).